### ISSUE TITLE
Criminal-tendency is calculated too early

### ISSUE THREAD

**Issue by mariopaolucci** [LABSS/PROTON-OC#219]
In the setup we have:

https://github.com/LABSS/PROTON-OC/blob/5e7e34ac6a8003bc8ddd157ea2e41b818236eb94/PROTON-OC.nlogo#L224-L226

But this means that the OC members don't get the OC multiplier, and that won't be corrected until the end of the first year. 

I'm asking @Kubasinska to check the numbers and, if this is the case, to repair the bug.

**Comment by Kubasinska** [comment]
Yes, but no! Is true that during setup the oc-member agents don't get their OC multiplier. The setup-oc-groups procedure assumes that the criminal-tendency has already been calculated, so we can't just move the procedure.
I checked the criminal tendency through this code to see the variation through ticks.

```logo
 show word "tick: " ticks 
  show (word "oc-member? = true: " "mean=" mean [criminal-tendency] of persons with [oc-member? = true] 
                                   "min=" min [criminal-tendency] of persons with [oc-member? = true]
                                   "max=" max [criminal-tendency] of persons with [oc-member? = true])
  
  show (word "oc-member? = false: " "mean=" mean [criminal-tendency] of persons with [oc-member? = false] 
                                   "min=" min [criminal-tendency] of persons with [oc-member? = false]
                                   "max=" max [criminal-tendency] of persons with [oc-member? = false])
  show "-----------------------------------"
```
At tick 12 I expected a leap in the criminal tendency, and indeed there is a variation but it seems too low and also, the two averages, should be more or less equal (even if the oc-members are definitely lower than the non oc-members) and they are not.

![image](https://user-images.githubusercontent.com/3855193/97679677-dcb17a00-1a95-11eb-9cf4-e6eb046ff00f.png)

The calculate-criminal-tendency procedure activates if tick modulo tick-per-year is equal to 0.
If we assume that tick = 1 and tick-per-year = 12, `12 - (1 * int(12/1))] = 0`. **This means that all procedures that are activated every tick-per-year are also activated at tick=1**. Therefore the criminal-tendency is correctly calculated at the beginning of the simulation.

**Comment by mariopaolucci** [comment]
it worked. We must change the comment..
```
 if ((ticks mod ticks-per-year) = 0) [ ; this should be 11, probably, otherwise
```
